rfcs/README.md

```diff
@@ -0,0 +1,153 @@
+itself or be linked as a reference.
+
+### What do you need to include in an RFC?
+
+We provide a [template][template] to help you create new RFCs. Be sure you
+clearly describe the problem that you are trying to solve in detail, including
+requirements and SLAs if warranted. It is impossible to meaningfully evaluate a
+proposed solution without this.
+
+There a number of sections which are required to move an RFC from Draft to
+Discussion status (i.e. to the point where it will be actively reviewed and
+discussed):
+
+- *Introduction* - One concise paragraph describing the proposal.
+- *Motivation & Scope* - Why are we doing this and what use cases does it support?
+- *Goals & Non-goals* - A short list of bullet points of what the goals of the
+  system are, and, sometimes more importantly, what the non-goals are. Non-goals
+  are things that could reasonably be goals, but that you're ruling out for
+  specific reasons.
+- *Design* - The trade-offs you made in considering the design of your software
+  and an explanation of a solution that best satisfies the goals you laid out
+  above.
+- *Alternatives* - What are other possible solutions and why did you rule them
+  out?
+- *Prior art* - How has this problem been solved or approached before?
+- *Unresolved questions* - What questions will be *eventually* be resolved through
+  the RFC process, as part of the specification, through the implementation
+  process, or only in future work?
+- *Impact & Concerns* - How will this have an impact beyond Fission (if it does),
+  and what cross-cutting concerns do we need to consider?
+
+The [template][template] includes these sections with more detail about how to
+address them.
+
+### On RFC Status
+
+An RFC moves through stages as part of its lifecycle. These are:
+
+- `Draft`: pre-discussion draft of the proposal. Our recommendation is you write
+  a short draft as quickly as possible and get your project's members to
+  help you shape it into a discussion-ready RFC.
+  Your RFC exists as a **draft PR** against `main`.
+- `Discussion`: active review and conversation. Your RFC exists as a
+  **ready to review PR** against `main`.
+- `Published`: the RFC has been accepted, merged into `main`, and is ready for
+  work.
+- `Abandoned`: the RFC had no activity for more than 30 days. The associated PR
+  will be marked `Abandoned`.
+- `Suspended`: during the RFC process, the engaged parties determined that
+  though the proposal might be a good idea, it's not the right time for it. The
+  PR can be marked `Suspended` and returned to later.
+- `Rejected`: a consensus was reached that this was not the right solution for
+  the problem. The associated PR will be marked `Rejected`.
+
+During the `Draft` and `Discussion` stages of an RFC, it may be helpful to write
+prototypes or gists to better convey a line of thinking.
+
+Once an RFC merged into `main`, it's considered `Published`.
```

Comment:
I'm less certain about this section for our uses. I see this a lot in community projects, but today this repo is mostly used for agreeing on how something work and we hammer out stuff in PR. That's not to say "no", but I mainly don't think we'll just not merge (or delete) rejected proposals

Reply:
makes sense.

Reply:
The draft/discussion stuff is obvious for sure, but it's good to just jot down for clarity (esp for new peeps joining).